modules/administration-guide/examples/che-prepulled-images-table.adoc

```diff
@@ -22,4 +22,26 @@
 |che-nodejs10-ubi |quay.io/eclipse/che-nodejs10-ubi:{prod-ver} |{prod-ver}
 
 |che-quarkus |quay.io/eclipse/che-quarkus:{prod-ver} |{prod-ver}
-|===
\ No newline at end of file
+|===
+
+[NOTE]
+====
+The table of recommended images to pre-pull is updated once per release and might vary from the most updated build at the time.
+
+To get the latest build:
+
+. Use the following scriptlet:
++
+[subs="+attributes,+quotes"]
+----
+$➔ curl -sSLo- https://raw.githubusercontent.com/eclipse/che-operator/master/olm/eclipse-che-preview-openshift/deploy/olm-catalog/eclipse-che-preview-openshift/{che-ver}/eclipse-che-preview-openshift.v{che-ver}.clusterserviceversion.yaml | yq -r '.spec.relatedImages[].image' | sort | uniq
+----
++
+. Exclude redundant, unused digests, and unsupported images for OpenShift pre-pulling such as:
```

Comment:
This instruction is not going to be something a user can obviously/easily do.

To make it easier, perhaps we should instead use:

```
$➔ curl -sSLo- https://raw.githubusercontent.com/eclipse/che-operator/master/olm/eclipse-che-preview-openshift/deploy/olm-catalog/eclipse-che-preview-openshift/{che-ver}/eclipse-che-preview-openshift.v{che-ver}.clusterserviceversion.yaml | yq -r '.spec.relatedImages[]
```

And then provide a snippet like:
```
...
{
  "name": "che-theia-7.19.2",
  "image": "quay.io/eclipse/che-theia@sha256:11b5fe33edd72df22f76d7933ba79029051f4c0c6adc6c1a91f2dfe4228455c9"
}
{
  "name": "che-theia-7.20.0",
  "image": "quay.io/eclipse/che-theia@sha256:92a8bac94e446f1f933d1a92bc96dace9450c9f4efdef5539a429a37d14be904"
}
{
  "name": "che-theia-7.20.1",
  "image": "quay.io/eclipse/che-theia@sha256:a56428ee28ceefca2a1cba51093f6015e0eee82659fec786f2e1fdc939fdb856"
}
...
```

With the instruction:

```
. Where multiple versions of the same image are listed, include only the latest one. 
. Exclude also the machine-exec image, as it cannot be pre-pulled with image-puller.

Note also that images without the `sleep` command and image that mount volumes cannot be pre-pulled, and thus need to be pulled by hand with `skopeo copy --all ...`.
```